safekeeper/src/metrics.rs

```diff
@@ -138,6 +157,15 @@ pub static BACKUP_ERRORS: Lazy<IntCounter> = Lazy::new(|| {
     )
     .expect("Failed to register safekeeper_backup_errors_total counter")
 });
+/* BEGIN_HADRON */
```

Comment:
do we still want to keep the codename around?

Reply:
yes.